MapSyncer/app.py

```diff
@@ -8,8 +8,8 @@
 import requests
 from flask import Flask, render_template, request, jsonify, redirect, url_for, session
 from mapilio_kit.base import authenticator
-from mapilio_kit.components.edit_config import edit_config
-from mapilio_kit.components.login import list_all_users
+from mapilio_kit.components.utilities.edit_config import edit_config
+from mapilio_kit.components.auth.login import list_all_users
```

Comment:
the code block is a duplicate

Reply:
the other pull request has been closed, that's why, it's necessary to have it here :tada: